### ISSUE TITLE
compile production env output [object Object] in css file

### ISSUE THREAD

**Issue by danielforsberg** [jenius/roots#667]
I have an `app.wp.coffee` with:

```javascript
axis         = require 'axis'
rupture      = require 'rupture'
autoprefixer = require 'autoprefixer-stylus'
js_pipeline  = require 'js-pipeline'
css_pipeline = require 'css-pipeline'
bootstrap	 = require 'bootstrap-styl'

module.exports =
	output: '../docroot/wp-content/themes/millennium/assets'

	ignores: ['readme.md', '**/layout.*', '**/_*', '.gitignore', 'ship.*conf', 'public']

	extensions: [
    js_pipeline(files: 'assets/js/*.js', out: 'js/build.js', minify: true),
    css_pipeline(files: 'assets/css/*.styl', out: 'css/build.css', minify: true)
  ]

	stylus:
    use: [axis(), rupture(), autoprefixer(), bootstrap()]

	jade:
		pretty: true

```

`roots compile -e wp` brings all the files where the output is set but the javascript is empty and the css file contains the text `[object Object]`

This worked before the update to 3.1.0

Thanks

**Comment by edravis** [comment]
I'm not sure if something changed with this new release but it has some very bad issues (see #654)
I'd recommend you to rollback to 3.1.0 (`npm i roots@3.1.0 -g`) and stay in there since the 3.2.0 became more stable. Anyway, you could wait for @jenius to shed some light.

**Comment by danielforsberg** [comment]
Thanks for the tip with the rollback, but wierdly I still get the same result. (Rolling back to `3.0.0`).

**Comment by johnpeele** [comment]
+1

I am getting this same issue. I am on `3.1.0` as well.

**Comment by alexsyi** [comment]
Hit this with a fresh `roots new`, too:

```
$ roots new foobar
? Your project's name: foobar
? A brief description of your project:
▸ project created
▸ dependencies installing
....
▸ dependencies finished installing
▸ project initialized at /private/tmp/foobar
▸ using template: roots-base
✓ done!

$ cd foobar
$ roots compile -e production
compiling... done!

$ cat public/css/build.1441a7909c087dbbe7ce59881b9df8b9.css
[object Object]

**Comment by brewingcode** [comment]
This is fixed in commit carrot/roots-css-pipeline@5c588a8b9b23b292f54c7fd89037dd92e127eaf0 (included in v0.3.0), so I've found it easiest to just use `0.3.x`  as my css-pipeline in my package.json:

```json
    "coffee-script": "1.9.x",
    "css-pipeline": "0.3.x",
    "jade": "1.x",
```

This produces properly minified css.

**Comment by danielforsberg** [comment]
Thanks @brewingcode - This resolved my issue
